pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7817033?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7817033?buildTab=artifacts#/) on master @ [ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410](https://github.com/cockroachdb/cockroach/commits/ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1411717807
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:117: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1411717807
--- FAIL: TestTenantStatusAPI (5.47s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.03s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1215: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2826/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1215
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.01s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot branch-release-22.2]
</p>
</details>
/cc @cockroachdb/sql-observability @cockroachdb/multi-tenant
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-22078

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8007779?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8007779?buildTab=artifacts#/) on master @ [93ed65565357538c9048ff45c878a493f2ed9b45](https://github.com/cockroachdb/cockroach/commits/93ed65565357538c9048ff45c878a493f2ed9b45):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI378325650
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:135: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI378325650
--- FAIL: TestTenantStatusAPI (4.24s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.17s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1233: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2801/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1233
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8015069?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8015069?buildTab=artifacts#/) on master @ [94e03c016955dfd64d4e15358ea92226a8f362aa](https://github.com/cockroachdb/cockroach/commits/94e03c016955dfd64d4e15358ea92226a8f362aa):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4245218489
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:135: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4245218489
--- FAIL: TestTenantStatusAPI (5.28s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.02s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1233: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2406/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1233
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8209008?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8209008?buildTab=artifacts#/) on master @ [642afd6e8c8d1f967da4aa0c3c08e2bdc0495100](https://github.com/cockroachdb/cockroach/commits/642afd6e8c8d1f967da4aa0c3c08e2bdc0495100):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3962141324
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:135: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3962141324
--- FAIL: TestTenantStatusAPI (4.27s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1233: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2500/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1233
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8367278?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8367278?buildTab=artifacts#/) on master @ [b6434c942b4811c088349c43a81a46a53fb9b6c0](https://github.com/cockroachdb/cockroach/commits/b6434c942b4811c088349c43a81a46a53fb9b6c0):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI320468523
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:143: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI320468523
--- FAIL: TestTenantStatusAPI (10.56s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.35s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1249: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3364/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1249
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #95341 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-sql-observability branch-release-22.2.2-with-dms-changes]
- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8440670?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8440670?buildTab=artifacts#/) on master @ [6eabc2f348710f4146ff984bc346cd8cbe96f83b](https://github.com/cockroachdb/cockroach/commits/6eabc2f348710f4146ff984bc346cd8cbe96f83b):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4136927739
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:143: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4136927739
--- FAIL: TestTenantStatusAPI (10.26s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.57s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1249: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2891/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1249
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #95341 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [C-test-failure O-robot T-sql-observability branch-release-22.2.2-with-dms-changes]
- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8580406?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8580406?buildTab=artifacts#/) on master @ [5fbcd8a8deac0205c7df38e340c1eb9692854383](https://github.com/cockroachdb/cockroach/commits/5fbcd8a8deac0205c7df38e340c1eb9692854383):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1216025310
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:144: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1216025310
--- FAIL: TestTenantStatusAPI (10.65s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.35s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1225: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1626/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1225
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8711026?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8711026?buildTab=artifacts#/) on master @ [64a867acd25c0a214209957eefb6483d1158b4f0](https://github.com/cockroachdb/cockroach/commits/64a867acd25c0a214209957eefb6483d1158b4f0):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4153507494
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:149: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4153507494
--- FAIL: TestTenantStatusAPI (18.18s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.16s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:211: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1612/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:211
        	Error:      	Not equal: 
        	            	expected: 45
        	            	actual  : 46
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.16s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8711024?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8711024?buildTab=artifacts#/) on master @ [64a867acd25c0a214209957eefb6483d1158b4f0](https://github.com/cockroachdb/cockroach/commits/64a867acd25c0a214209957eefb6483d1158b4f0):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI218717896
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:149: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI218717896
--- FAIL: TestTenantStatusAPI (10.66s)
=== RUN   TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
    tenant_status_test.go:1295: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2210/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1295
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination
        --- FAIL: TestTenantStatusAPI/tenant_ranges/test_tenant_ranges_pagination (0.00s)
=== RUN   TestTenantStatusAPI/tenant_ranges
    --- FAIL: TestTenantStatusAPI/tenant_ranges (0.18s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8724617?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/8724617?buildTab=artifacts#/) on master @ [a0ab818e89508ca0b65926a4faac4c563d114acf](https://github.com/cockroachdb/cockroach/commits/a0ab818e89508ca0b65926a4faac4c563d114acf):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI285013507
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:149: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/7067ad80d000d85ce2eeee021e911c58/logTestTenantStatusAPI285013507
--- FAIL: TestTenantStatusAPI (10.86s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.04s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:211: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/7625/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:211
        	Error:      	Not equal: 
        	            	expected: 46
        	            	actual  : 48
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.04s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8726063?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8726063?buildTab=artifacts#/) on master @ [af3bf237ce3774258a13a0c2aa0e2da22716270f](https://github.com/cockroachdb/cockroach/commits/af3bf237ce3774258a13a0c2aa0e2da22716270f):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3909493003
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:149: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3909493003
--- FAIL: TestTenantStatusAPI (10.37s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.02s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:211: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2610/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:211
        	Error:      	Not equal: 
        	            	expected: 48
        	            	actual  : 49
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.02s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8726065?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8726065?buildTab=artifacts#/) on master @ [af3bf237ce3774258a13a0c2aa0e2da22716270f](https://github.com/cockroachdb/cockroach/commits/af3bf237ce3774258a13a0c2aa0e2da22716270f):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2297664063
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:149: -- test log scope end --
--- FAIL: TestTenantStatusAPI (18.91s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.17s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:211: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1792/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:211
        	Error:      	Not equal: 
        	            	expected: 45
        	            	actual  : 46
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2297664063
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.17s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8740552?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8740552?buildTab=artifacts#/) on master @ [7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260](https://github.com/cockroachdb/cockroach/commits/7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4000609239
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4000609239
--- FAIL: TestTenantStatusAPI (21.99s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.19s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3213/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 706
        	            	actual  : 705
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.19s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8750679?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8750679?buildTab=artifacts#/) on master @ [c95bef097bd4c213c6b5c0c125a9a846c4479d73](https://github.com/cockroachdb/cockroach/commits/c95bef097bd4c213c6b5c0c125a9a846c4479d73):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI293104222
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI293104222
--- FAIL: TestTenantStatusAPI (31.25s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.48s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2832/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 702
        	            	actual  : 720
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.48s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8758030?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8758030?buildTab=artifacts#/) on master @ [3d054f37c7c87f53cb56fac4e5500f0d1130d09a](https://github.com/cockroachdb/cockroach/commits/3d054f37c7c87f53cb56fac4e5500f0d1130d09a):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2051720917
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
--- FAIL: TestTenantStatusAPI (28.42s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.35s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1466/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 700
        	            	actual  : 718
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.35s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8764488?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8764488?buildTab=artifacts#/) on master @ [e9c96e7179e19aae2f8d386f67eb950db8c3354b](https://github.com/cockroachdb/cockroach/commits/e9c96e7179e19aae2f8d386f67eb950db8c3354b):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4194040022
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4194040022
--- FAIL: TestTenantStatusAPI (32.55s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.96s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1466/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 703
        	            	actual  : 721
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.95s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8803931?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8803931?buildTab=artifacts#/) on master @ [e028ce5b14505dfd17ef8b13001c0ab8ac811e3c](https://github.com/cockroachdb/cockroach/commits/e028ce5b14505dfd17ef8b13001c0ab8ac811e3c):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2463712998
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2463712998
--- FAIL: TestTenantStatusAPI (10.37s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.02s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2953/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 704
        	            	actual  : 703
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.02s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8836820?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8836820?buildTab=artifacts#/) on master @ [fca69b046f6389fbc4d2b6db428e27a5d7277773](https://github.com/cockroachdb/cockroach/commits/fca69b046f6389fbc4d2b6db428e27a5d7277773):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2175421116
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2175421116
--- FAIL: TestTenantStatusAPI (22.17s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.16s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1449/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 706
        	            	actual  : 705
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.16s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8844191?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8844191?buildTab=artifacts#/) on master @ [39c06b5a438c01c93ffbfeeefe702d3f9b620eaf](https://github.com/cockroachdb/cockroach/commits/39c06b5a438c01c93ffbfeeefe702d3f9b620eaf):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI224365972
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI224365972
--- FAIL: TestTenantStatusAPI (23.10s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.20s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1484/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 701
        	            	actual  : 700
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.20s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8881290?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8881290?buildTab=artifacts#/) on master @ [21786aa112e6b822858f281c1cc59608987c5c0a](https://github.com/cockroachdb/cockroach/commits/21786aa112e6b822858f281c1cc59608987c5c0a):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI259387082
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI259387082
--- FAIL: TestTenantStatusAPI (10.99s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.04s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3706/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 699
        	            	actual  : 698
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.04s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8894402?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8894402?buildTab=artifacts#/) on master @ [3036617a2c2f1e9c694857cf02b34e5ce8de5b29](https://github.com/cockroachdb/cockroach/commits/3036617a2c2f1e9c694857cf02b34e5ce8de5b29):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3751351963
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3751351963
--- FAIL: TestTenantStatusAPI (28.31s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2841/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 704
        	            	actual  : 722
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.31s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.31s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8894400?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8894400?buildTab=artifacts#/) on master @ [3036617a2c2f1e9c694857cf02b34e5ce8de5b29](https://github.com/cockroachdb/cockroach/commits/3036617a2c2f1e9c694857cf02b34e5ce8de5b29):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3185017626
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3185017626
--- FAIL: TestTenantStatusAPI (10.58s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.05s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2226/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 703
        	            	actual  : 702
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.05s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8907296?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8907296?buildTab=artifacts#/) on master @ [8c54290f5f1ad68976e59a5f623126ae9d734bf0](https://github.com/cockroachdb/cockroach/commits/8c54290f5f1ad68976e59a5f623126ae9d734bf0):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3228391
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3228391
--- FAIL: TestTenantStatusAPI (10.71s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.03s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/21/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 704
        	            	actual  : 703
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8933318?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8933318?buildTab=artifacts#/) on master @ [14b43be03c1c246765be17aa15245493a42c401d](https://github.com/cockroachdb/cockroach/commits/14b43be03c1c246765be17aa15245493a42c401d):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2808844470
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2808844470
--- FAIL: TestTenantStatusAPI (28.86s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4099/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 703
        	            	actual  : 721
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.38s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.38s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8948842?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8948842?buildTab=artifacts#/) on master @ [43a25a09f92408accdeb3cfed1eb35e02d375877](https://github.com/cockroachdb/cockroach/commits/43a25a09f92408accdeb3cfed1eb35e02d375877):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3920764942
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3920764942
--- FAIL: TestTenantStatusAPI (10.82s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.02s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3073/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 702
        	            	actual  : 701
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.02s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8965590?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8965590?buildTab=artifacts#/) on master @ [1b162d1b274eec7b307fbbfca7294460bfdef025](https://github.com/cockroachdb/cockroach/commits/1b162d1b274eec7b307fbbfca7294460bfdef025):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2001275349
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2001275349
--- FAIL: TestTenantStatusAPI (34.96s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.26s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3333/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 724
        	            	actual  : 718
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.26s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8998083?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8998083?buildTab=artifacts#/) on master @ [01e14f68f7c757cab1c878cebfecd73c5119046b](https://github.com/cockroachdb/cockroach/commits/01e14f68f7c757cab1c878cebfecd73c5119046b):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2078033790
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2078033790
--- FAIL: TestTenantStatusAPI (29.81s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.70s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3526/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 723
        	            	actual  : 717
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.70s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012281?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012281?buildTab=artifacts#/) on master @ [5b2a5670cbbe895d76602c230390816e783e0caa](https://github.com/cockroachdb/cockroach/commits/5b2a5670cbbe895d76602c230390816e783e0caa):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI610391313
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI610391313
--- FAIL: TestTenantStatusAPI (30.93s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.51s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4439/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 710
        	            	actual  : 728
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.51s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9020198?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9020198?buildTab=artifacts#/) on master @ [95684388bfbb372ac8c7e1a8ebd4a0d6f447e147](https://github.com/cockroachdb/cockroach/commits/95684388bfbb372ac8c7e1a8ebd4a0d6f447e147):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1983542420
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1983542420
--- FAIL: TestTenantStatusAPI (35.79s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.66s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2852/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 728
        	            	actual  : 722
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.66s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9030052?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9030052?buildTab=artifacts#/) on master @ [e4924e2b9be4a36d466beab53a80df9241df4783](https://github.com/cockroachdb/cockroach/commits/e4924e2b9be4a36d466beab53a80df9241df4783):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4233483267
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI4233483267
--- FAIL: TestTenantStatusAPI (31.31s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.52s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:222: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1455/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:222
        	Error:      	Not equal: 
        	            	expected: 701
        	            	actual  : 719
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.52s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9051192?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9051192?buildTab=artifacts#/) on master @ [43f7cde6f036f720388912bde7310095c2f8ca9e](https://github.com/cockroachdb/cockroach/commits/43f7cde6f036f720388912bde7310095c2f8ca9e):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2655711184
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:159: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2655711184
--- FAIL: TestTenantStatusAPI (41.49s)
=== RUN   TestTenantStatusAPI/txn_id_resolution
=== CONT  TestTenantStatusAPI/txn_id_resolution
    tenant_status_test.go:1245: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1706/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1245
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1706/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/soon.go:71
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1706/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/retry.go:213
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1706/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/soon.go:77
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1706/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/soon.go:59
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1706/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/soon.go:41
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1706/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1232
        	            				/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/1706/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:1263
        	Error:      	Should not be: 0x0
        	Test:       	TestTenantStatusAPI/txn_id_resolution
    --- FAIL: TestTenantStatusAPI/txn_id_resolution (0.07s)
=== RUN   TestTenantStatusAPI/txn_id_resolution/tenant_cluster
=== CONT  TestTenantStatusAPI/txn_id_resolution/tenant_cluster
    testing.go:1343: test executed panic(nil) or runtime.Goexit: subtest may have called FailNow on a parent test
        --- FAIL: TestTenantStatusAPI/txn_id_resolution/tenant_cluster (0.03s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9071162?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9071162?buildTab=artifacts#/) on master @ [9b43dc50d468171bc9919dc684b7274b63e13da2](https://github.com/cockroachdb/cockroach/commits/9b43dc50d468171bc9919dc684b7274b63e13da2):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI2258820115
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:141: -- test log scope end --
--- FAIL: TestTenantStatusAPI (38.53s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.94s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:204: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3421/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/serverccl/statusccl/statusccl_test_/statusccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/serverccl/statusccl/tenant_status_test.go:204
        	Error:      	Not equal: 
        	            	expected: 803
        	            	actual  : 800
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.93s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131665?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131665?buildTab=artifacts#/) on master @ [53dbb86acb1d48309530181b94838faf937084d3](https://github.com/cockroachdb/cockroach/commits/53dbb86acb1d48309530181b94838faf937084d3):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1655917537
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:141: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI1655917537
--- FAIL: TestTenantStatusAPI (10.89s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:204: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/statusccl_test/pkg/ccl/serverccl/statusccl/tenant_status_test.go:204
        	Error:      	Not equal: 
        	            	expected: 806
        	            	actual  : 805
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.06s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.06s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9142123?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9142123?buildTab=artifacts#/) on master @ [9acc75317aebfdfe69fb097a8a28e0040c7a67fd](https://github.com/cockroachdb/cockroach/commits/9acc75317aebfdfe69fb097a8a28e0040c7a67fd):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3215470422
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:141: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3215470422
--- FAIL: TestTenantStatusAPI (48.40s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:204: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/statusccl_test/pkg/ccl/serverccl/statusccl/tenant_status_test.go:204
        	Error:      	Not equal: 
        	            	expected: 826
        	            	actual  : 825
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.22s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.22s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/serverccl/statusccl/statusccl_test.TestTenantStatusAPI [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162737?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162737?buildTab=artifacts#/) on master @ [b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0](https://github.com/cockroachdb/cockroach/commits/b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0):


```
=== RUN   TestTenantStatusAPI
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3686556355
=== CONT  TestTenantStatusAPI
    tenant_status_test.go:141: -- test log scope end --
CREATE UNIQUE INDEX test_pkey ON test_db1.public.test USING btree (k ASC)test logs left over in: /artifacts/tmp/_tmp/8bc2de686563d04c3522b0df244fc129/logTestTenantStatusAPI3686556355
--- FAIL: TestTenantStatusAPI (50.81s)
=== RUN   TestTenantStatusAPI/tenant_span_stats
    --- FAIL: TestTenantStatusAPI/tenant_span_stats (0.30s)
=== RUN   TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
    tenant_status_test.go:204: 
        	Error Trace:	pkg/ccl/serverccl/statusccl/statusccl_test/pkg/ccl/serverccl/statusccl/tenant_status_test.go:204
        	Error:      	Not equal: 
        	            	expected: 822
        	            	actual  : 821
        	Test:       	TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out
        --- FAIL: TestTenantStatusAPI/tenant_span_stats/test_KV_node_fan-out (0.30s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92382 pkg/ccl/serverccl/statusccl/statusccl_test: TestTenantStatusAPI failed [A-observability-inf C-test-failure O-robot T-observability-inf branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStatusAPI.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/serverccl/statusccl/tenant_status_test.go
+++ b/pkg/ccl/serverccl/statusccl/tenant_status_test.go
@@ -1290,8 +1290,6 @@ func testTenantRangesRPC(_ context.Context, t *testing.T, helper serverccl.Tenan
 	})
 
 	t.Run("test tenant ranges pagination", func(t *testing.T) {
-		skip.WithIssue(t, 92979,
-			"flaky test, difficult to reproduce locally. Skip until resolved.")
 		ctx := context.Background()
 		resp1, err := tenantA.TenantRanges(ctx, &serverpb.TenantRangesRequest{
 			Limit: 1,
@@ -1302,18 +1300,34 @@ func testTenantRangesRPC(_ context.Context, t *testing.T, helper serverccl.Tenan
 			require.Len(t, ranges.Ranges, 1)
 		}
 
-		resp2, err := tenantA.TenantRanges(ctx, &serverpb.TenantRangesRequest{
-			Limit:  1,
-			Offset: resp1.Next,
+		sql := helper.TestCluster().TenantConn(0)
+		// Wait for the split queue to process some before requesting the 2nd range.
+		// We expect an offset for the 3rd range, so wait until at least 3 ranges exist.
+		testutils.SucceedsSoon(t, func() error {
+			res := sql.QueryStr(t, "SELECT count(*) FROM crdb_internal.ranges")
+			require.Equal(t, len(res), 1)
+			require.Equal(t, len(res[0]), 1)
+			rangeCount, err := strconv.Atoi(res[0][0])
+			require.NoError(t, err)
+			if rangeCount < 3 {
+				return errors.Newf("expected >= 3 ranges, got %d", rangeCount)
+			}
+
+			resp2, err := tenantA.TenantRanges(ctx, &serverpb.TenantRangesRequest{
+				Limit:  1,
+				Offset: resp1.Next,
+			})
+			require.NoError(t, err)
+			require.Equal(t, 2, int(resp2.Next))
+			for locality, ranges := range resp2.RangesByLocality {
+				require.Len(t, ranges.Ranges, 1)
+				// Verify pagination functions based on ascending RangeID order.
+				require.True(t,
+					resp1.RangesByLocality[locality].Ranges[0].RangeID < ranges.Ranges[0].RangeID)
+			}
+			return nil
 		})
-		require.NoError(t, err)
-		require.Equal(t, 2, int(resp2.Next))
-		for locality, ranges := range resp2.RangesByLocality {
-			require.Len(t, ranges.Ranges, 1)
-			// Verify pagination functions based on ascending RangeID order.
-			require.True(t,
-				resp1.RangesByLocality[locality].Ranges[0].RangeID < ranges.Ranges[0].RangeID)
-		}
+
 	})
 }
 

